src/syscall/syscall_unix.go

```diff
@@ -16,3 +19,8 @@ type SockaddrInet6 struct {
 	Addr   [16]byte
 	raw    RawSockaddrInet6
 }
+
+const (
+	SYS_FORK   = 57
+	SYS_EXECVE = 59
+)
```

Comment:
These can be removed, because every actual operating system we have (Linux, MacOS, Windows) doesn't use the TinyGo syscall package.
(If removing these leads to compilation errors, you're probably using them from something that's not an operating system and probably need to adjust build tags at the call site).